generator/src/test/java/com/scottlogic/deg/generator/builders/ConstraintNodeBuilderDepreciated.java

```diff
@@ -25,44 +25,45 @@
 import java.util.List;
 import java.util.Set;
 
-public class ConstraintNodeBuilder {
+// Please use ConstraintNodeBuilder in main instead of this test builder
```

Comment:
thinking about it, this one is in the test folder, not the main folder, so can't be used in main anyway.

maybe testConstraintNodeBuilder rather than ConstraintNodeBuilderDepricated 

/shrug

Reply:
Yeah. As discussed there are a few methods which are useful for tests on the old builder